app/Services/AssignUserToOrganizationsService.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Services;
+
+use App\Models\Organization;
+use App\Models\User;
+use Illuminate\Support\Facades\Http;
+
+class AssignUserToOrganizationsService
+{
+    public function assign(): void
+    {
+        $user = User::query()->where("id", auth()->user()->id)->first();
+
+        $response = Http::withHeaders([
+            "Authorization" => "Bearer " . $user->github_token,
+        ])->get("https://api.github.com/user/orgs");
```

Comment:
What about handling error response or non-functional service errors?
Maybe move that action to new service - good package for that: https://docs.saloon.dev/

Reply:
used saloon for sending request and error response handling

Reply:
Where are u handling errors?